pkg/process/config/config_test.go

```diff
@@ -486,27 +424,6 @@ func TestEnvOrchestratorAdditionalEndpoints(t *testing.T) {
 	}
 }
 
-func TestEnvAdditionalEndpointsMalformed(t *testing.T) {
-	newConfig()
-	defer restoreGlobalConfig()
-
-	assert := assert.New(t)
-
-	expected := make(map[string]string)
-	expected["apikey_20"] = "my-process-app.datadoghq.com" // from config file
-
-	os.Setenv("DD_PROCESS_ADDITIONAL_ENDPOINTS", `"https://url1.com","key1"`)
-	defer os.Unsetenv("DD_PROCESS_ADDITIONAL_ENDPOINTS")
```

Comment:
Are we testing cases with malformed API endpoints as well in the new tests?

Reply:
Yep! If one of the API endpoints is not a valid url, we just return an error.